### PR TITLE
SEO updates for Version

### PR DESCRIPTION
**Pull request by Thraka**
Contributes to #9440

### ALERT
I deleted `docs/core/versions/install-management.md` and redirected to the how to remove sdk/runtime article. This article seemed pointless and just reiterated information in the how to remove article. It also had < 500 page views.

### CODE REVIEW

**Comment by mairaw** [comment]
Wondering if parts of that topics then should move into the article you're redirecting it to.

**Comment by Thraka** [comment]
@mairaw I think maybe porting 

>In general, you only need the latest SDK and latest patch version of the runtimes required for your application. Instances where retaining older SDK or Runtime versions include maintaining project.json-based applications. Unless your application has specific reasons for earlier SDKs or runtimes, you may safely remove older versions.

would be handy

**Review by BillWagner — CHANGES_REQUESTED** [review]
Overall, this LGTM.  I had one comment for an edit before you :shipit:.

File: docs/core/versions/install-management.md [hunk 1]
```diff
@@ -1,18 +0,0 @@
...
-ms.author: wiwagn
-ms.date: 08/22/2018
----
-# Manage .NET Core installations
-
-.NET Core allows multiple versions of the SDK and Runtime to exist side-by-side, enabling version dependency flexibility for both building and running .NET Core applications. These installation and automatic version roll-forward behaviors offer valuable benefits, but one pronounced drawback. As updates of the .NET Core SDK or .NET Core Runtime are installed, previous versions remain on-disk. Multiple installed versions increase disk usage over time. More than 50 .NET Core SDK updates have been released. There may be many versions of the SDK and Runtime installed on your system that could be removed.
-
-## Safe to remove?
-
-The [.NET Core version selection](selection.md) behaviors and the runtime compatibility of .NET Core across updates enables safe removal of previous versions. .NET Core runtime updates are compatible within a major version 'band' such as 1.x and 2.x. Additionally, newer releases of the .NET Core SDK generally maintain the ability to build applications that target previous versions of the runtime in a compatible manner.
```
[the remark this answers is not in the document]
Comment (Thraka) [reply docs/core/versions/install-management.md]: OK I moved this into the remove article.

**Pull request merged**